### PR TITLE
add TargetCluster URL in InternalWorkspace's status

### PR DESCRIPTION
**Pull request by filariow**
requires #308

### CODE REVIEW

**Review by sadlerap — DISMISSED** [review]

File: e2e/step/workspace/workspace_then.go [hunk 1]
```diff
@@ -3,18 +3,52 @@ package workspace
...
+		if err != nil {
+			return false, err
+		}
+
+		w := restworkspacesv1alpha1.Workspace{}
+		k := types.NamespacedName{
+			Name:      workspacesv1alpha1.DisplayNameDefaultWorkspace,
+			Namespace: u.Status.CompliantUsername,
+		}
+		log.Printf("workspace key: %v", k)
```
Comment (filariow) [inline e2e/step/workspace/workspace_then.go:43]: remove this

File: e2e/pkg/poll/poll.go [hunk 2]
```diff
@@ -42,3 +43,27 @@ func getDurationFromEnv(envVar string, defaultDuration time.Duration) time.Durat
...
+// WaitForConditionImmediatelyJoiningErrors runs the function `condition` periodically to poll the
+// status of a condition.  It waits for either condition's first return value
+// to be `true`, or for a timeout to be hit.
+//
+// By default, this timeout is 1 minute, and `condition` is checked every
+// second.  These can be overridden with the `E2E_POLL_TIMEOUT` and
+// `E2E_POLL_STEP_DURATION` environment variables.
+//
+// The errors returned by the invocations of the `condition` function are handled and
+// and used in to return a Joined error if timeout is hit.
```
Comment (filariow) [inline e2e/pkg/poll/poll.go:56]:
```suggestion
// The errors returned by the invocations of the `condition` function are collected and
// -if timeout is hit- returned as a Joined error.
```

**Review by sadlerap — APPROVED** [review]

**Comment by sadlerap** [comment]
With those changes you've suggested, lgtm

**Pull request merged**